pywwt/annotation.py

```diff
@@ -132,13 +140,19 @@ class Polygon(Annotation):
 
     shape = 'polygon'
     """
-    The name of the shape (:class:`str`)
+    The name of the shape (:class:`str`).
     """
 
-    fill = Bool(False, help='Whether or not the polygon should be filled (:class:`bool`)').tag(wwt='fill')
-    fill_color = ColorWithOpacity('white', help='Assigns fill color for the polygon (:class:`str` or `tuple`)').tag(wwt='fillColor')
-    line_color = Color('white', help='Assigns line color for the polygon (:class:`str` or `tuple`)').tag(wwt='lineColor')
-    line_width = AstropyQuantity(1 * u.pixel, help='Assigns line width in pixels (:class:`~astropy.units.Quantity`)').tag(wwt='lineWidth')
+    fill = Bool(False, help='Whether or not the polygon should be filled '
+                            '(`bool`)').tag(wwt='fill')
+    fill_color = ColorWithOpacity(
+        'white', help='Assigns fill color for the polygon '
+                      '(`str` or `tuple`)').tag(wwt='fillColor')
+    line_color = Color('white', help='Assigns line color for the polygon '
+                                     '(`str` or `tuple`)').tag(wwt='lineColor')
+    line_width = AstropyQuantity(
+        1*u.pixel, help='Assigns line width in pixels '
+                        '(:class:`~astropy.units.Quantity`)').tag(wwt='lineWidth')
```

Comment:
I think this would be fine as:

```
line_width = AstropyQuantity(1*u.pixel,
                             help='Assigns line width in pixels '
                                  '(:class:`~astropy.units.Quantity`)').tag(wwt='lineWidth')
```

Many projects don't actually respect the 80 character recommendation, instead choosing something more pragmatic such as 100, so I think we should aim for that instead (the above is 93)

Reply:
In general I'm not a fan of putting anything on the left of the initial ( for readability - but at the end of the day, it doesn't *really* matter, so i'll leave it up to you :)